spec.html

```diff
@@ -5219,6 +5219,17 @@
             </tr>
             <tr>
               <td>
+                [[GlobalThisValue]]
+              </td>
+              <td>
+                Object
+              </td>
+              <td>
+                The value returned by `this` in global scope. This may be the same as the global object, or may not. For example, in HTML the `this` value is a `WindowProxy` object, whereas the global object is a `Window`.
```

Comment:
This was kind of in the spirit of "for example, in the HTML document object model the window property of the global object is the global object itself." Happy to drop it, but it seems kind of nice. Also happy to do require it to be an Object, although that is implicitly required by InitializeHostDefinedRealm's latest definition. (In general these descriptions seem non-normative to me...)